lib/src/test/scala/code/winitzki/jc/JoinRunBlockingSpec.scala

```diff
@@ -23,7 +23,7 @@ class JoinRunBlockingSpec extends FlatSpec with Matchers with TimeLimitedTests w
     tp0.shutdownNow()
   }
 
-  val timeLimit = Span(1000, Millis)
+  val timeLimit = Span(1500, Millis)
```

Comment:
you may need a Configuration module and specify such data at run-time externally. I  only know Play's HOCON Configuration, not sure what's appropriate for a thin library project.